db/migrate/20230726101028_create_posts.rb

```diff
@@ -2,7 +2,7 @@ class CreatePosts < ActiveRecord::Migration[7.0]
   def change
     create_table :posts do |t|
       t.string :name
-      t.string :status
+      t.string :status, default: "pending" 
```

Comment:
This migration is already run in the DB, do you think this way it will override the current schema?